helm/alfresco-identity-service/values.yaml

```diff
@@ -7,14 +7,23 @@ ingressHostName: "your ingress dns name"
 keycloak:
   rbac:
     create: false
-  secretName: none
-  secretKey: config
   keycloak:
     image:
       tag: 3.4.3.Final
       pullPolicy: IfNotPresent
     username: admin
     password: admin
+    extraVolumes: |
+      - name: realm-secret
+        secret:
+          secretName: realm-secret    
+    extraVolumeMounts: |
+      - name: realm-secret
+        mountPath: "/realm/"
+        readOnly: true
+    extraArgs: "-Dkeycloak.import=/realm/alfresco-realm.json"
```

Comment:
👍 for this approach to importing the realm

Reply:
It's a new feature added by Ryan Dawson to the official chart! 👍

Reply:
+1